CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
 cmake_minimum_required(VERSION 3.6)
```

Comment:
@mgallien While I was testing I noticed that there is still a problem. Not doing a clean build bite me again. I reverted the last commit and put the `include(ECMEnableSanitizers)` statement in `src/CMakeLists.txt` instead of `CMakeLists.txt`. Now the sanitizers get only enabled for gui, libsync and sync. Could you do a (hopefully) last test and confirm that it is working? Thanks:)

Reply:
this is indeed working for me but you lost the modification to enable the sanitizers in drone